pulp_smash/tests/pulp3/utils.py

```diff
@@ -290,3 +290,34 @@ def get_repo_versions(repo):
         [version['_href'] for version in versions],
         key=lambda url: int(urlsplit(url).path.split('/')[-2])
     )
+
+
+def get_artifact_paths(repo, version_href=None):
+    """Return the paths of artifacts present in a given repository version.
+
+    :param repo: A dict of information about the repository.
+    :returns: A list with the paths of units present in a given repository.
+    """
+    return set([
```

Comment:
Rather than creating a list and a set, create just a set:

```python
return {content_unit['artifact'] for content_unit in ...}
```

Set comprehension and dict comprehension syntax is similar. Here's samples of dict comprehension syntax:

```python
{key: val for key, val in ((1, 2), (3, 4), (5, 6))}
{item[0]: item[1] for item in ((1, 2), (3, 4), (5, 6))}
```